src/pages/management/webhooks/index.mdx

```diff
@@ -15,7 +15,7 @@ In orderd to receive webhooks, you need to [register](/management/configuration-
 
 - [See webhook methods.](/management/configuration-api/v3.4/#webhooks)
 - [Learn how to use webhooks with the LiveChat API.](https://developers.livechat.com/updates/setting-up-webhooks-with-livechat-api/)
-- [Read the Changelog.](../changelog/)
+- [Read the Changelog.](../management/changelog/)
```

Comment:
a tu Cię mam :D "kropeczki" działają tak, że przenoszą Cię w hierarchii plików "folder" wyżej > czyli w tym scenariuszu, wcześniej np. link wychodził do management i z management do docs/changelog (bo to powiedziały mu dwie kropeczki w "górę")
a teraz link wychodzi jeszcze gdzieś indziej, z management do docs i z docs do czeluści internetu 😆 ale można to rozwiązać tak, że:
1. damy jedną kropeczkę, czyli (./changelog) - wtedy link wyjdzie folder w górę od odnośnika, czyli do management/webhooks
2. nie damy kropeczek i zrobimy bezpośrednio (/management/changelog) - jestem za tą opcją bo gdybyśmy np cos zmieniali w przyszłości w dokumentacji miejscami itp to będzie nam łatwiej znaleźć ten link i dokąd on prowadzi i miał prowadzić docelowo :D

Reply:
jakbyś nie do końca rozumiala tego konceptu z kropeczkami to daj znać możemy się zdzwonić i postaram się Ci to pokazać hierarchicznie na plikach